plugin/trino-phoenix5/src/main/java/io/trino/plugin/phoenix5/MetadataUtil.java

```diff
@@ -15,26 +15,26 @@
 
 import org.apache.phoenix.util.SchemaUtil;
 
-import java.util.Optional;
+import javax.annotation.Nullable;
 
 import static io.trino.plugin.phoenix5.PhoenixMetadata.DEFAULT_SCHEMA;
 
 public final class MetadataUtil
 {
     private MetadataUtil() {}
 
-    public static String getEscapedTableName(Optional<String> schema, String table)
+    public static String getEscapedTableName(@Nullable String schema, String table)
     {
-        return SchemaUtil.getEscapedTableName(toPhoenixSchemaName(schema).orElse(null), table);
+        return SchemaUtil.getEscapedTableName(toPhoenixSchemaName(schema), table);
     }
 
-    public static Optional<String> toPhoenixSchemaName(Optional<String> trinoSchemaName)
+    public static @Nullable String toPhoenixSchemaName(@Nullable String trinoSchemaName)
     {
-        return trinoSchemaName.map(schemaName -> DEFAULT_SCHEMA.equalsIgnoreCase(schemaName) ? "" : schemaName);
+        return DEFAULT_SCHEMA.equalsIgnoreCase(trinoSchemaName) ? "" : trinoSchemaName;
     }
 
-    public static Optional<String> toTrinoSchemaName(Optional<String> phoenixSchemaName)
+    public static @Nullable String toTrinoSchemaName(@Nullable String phoenixSchemaName)
     {
-        return phoenixSchemaName.map(schemaName -> schemaName.isEmpty() ? DEFAULT_SCHEMA : schemaName);
+        return phoenixSchemaName.isEmpty() ? DEFAULT_SCHEMA : phoenixSchemaName;
```

Comment:
`phoenixSchemaName.isEmpty()` may NPE. Also: shouldn't it compare against `NULL_SCHEMA_NAME`?